flang/test/Lower/target-features.f90

```diff
@@ -0,0 +1,22 @@
+! RUN: %flang_fc1 -emit-fir %s -o - | FileCheck %s --check-prefixes=ALL,NONE
```

Comment:
You should add `REQUIRES: amdgpu-registered-target`

Maybe we also should have one or another of these that check on x86 or Arm, to reduce the risk of accidental breaks. Unfortunately, all these tests are by nature architecture-dependent, so can't be made generic.

Reply:
Thanks for pointing this out. I added a similar x86 test with explicit -target-feature options added to the frontend invocations, rather than the implicit ones added for AMDGPU.